webpack/components/SCCProductPage/SCCProductPageActions.js

```diff
@@ -0,0 +1,27 @@
+import { API_OPERATIONS, put } from 'foremanReact/redux/API';
+import { translate as __ } from 'foremanReact/common/I18n';
+
+export const subscribeProductsWithReposAction = (
+  sccAccountId,
+  sccProductData,
+  handleSubscription,
+  sccSubscriptionData
+) =>
+  put({
+    type: API_OPERATIONS.PUT,
+    key: `subscribe_key_${sccAccountId}_${sccProductData[0].scc_product_id}_${sccProductData[0].repository_list}`,
+    url: `/api/scc_accounts/${sccAccountId}/bulk_subscribe_with_repos`,
+    params: { scc_product_data: sccProductData },
+    errorToast: (error) => __('Starting the subscription task failed.'),
+    handleSuccess: (response) =>
+      handleSubscription(response.data.id, sccSubscriptionData),
+  });
```

Comment:
if it helps, you can also `useAPI` hook from Foreman,
and use the hook directly inside the component,
this may look as:
```js
    const url = `/api/scc_accounts/${sccAccountId}/bulk_subscribe_with_repos`,
    const params: { scc_product_data: sccProductData },
    const errorToast: (error) => __('Starting the subscription task failed.'),
    const handleSuccess: (response) =>
      handleSubscription(response.data.id, sccSubscriptionData),
    const { response, status } = useAPI('put', url, { params, errorToast, handleSuccess });
```